internal/blockchain/chain.go

```diff
@@ -2371,6 +2372,15 @@ func New(ctx context.Context, config *Config) (*BlockChain, error) {
 		return nil, err
 	}
 
+	// Convert the minimum known work to a uint256 when it it exists.  Ideally,
```

Comment:
s / ... when it it exists. / ... when it exits.

Reply:
Done.